hack/boilerplate/boilerplate.generatego.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
generatego is not a term used elsewhere. Better maybe: generated_files or code-generator.

Reply:
How about boilerplate.generated_files.txt? and it only used to generated go files.